packages/terra-outline-view-table/package.json

```diff
@@ -36,7 +36,7 @@
     "prop-types": "^15.5.8",
     "terra-content-container": "^3.38.0",
     "terra-mixins": "^1.40.0",
-    "terra-table": "^4.34.0"
+    "terra-table": "^4.35.0"
```

Comment:
As terra-table v 4.35.0 is not released yet, I guess it needs to be "terra-table": "^4.34.0"

Reply:
That's correct !! missed it while selecting the components from releases 
Removed : https://github.com/cerner/terra-core/pull/3824/commits/c8f02479382460a78fe71f59eb170c3d5966109f